pandas/tests/util/test_util.py

```diff
@@ -76,3 +76,16 @@ def test_rng_context():
         with tm.RNGContext(1):
             assert np.random.randn() == expected1
         assert np.random.randn() == expected0
+
+
+def test_external_error_raised():
+    # Raising an external error message
+    with tm.external_error_raised(TypeError):
+        pytest.approx()
```

Comment:
I appreciate the attempt to try to trigger an actual external error message, but I think doing a `raise TypeError("test")` will suffice for our purposes.  `pytest.approx` is more vulnerable to unrelated failures (e.g. `approx` suddenly gets dropped from the `pytest` API).

Reply:
Haven't thought of that, thank you for letting me know :)